index.html

```diff
@@ -10,8 +10,18 @@
     <header>
       <div id="welcome">Welcome</div>
       <nav id="nav">
-        <a href="#"> <img src="./images/hamburger.png" alt="menu-button" /></a>
+        <a href="#" class="menu-toggle"> <img src="./images/hamburger.png" alt="menu-button" /></a>
       </nav>
+      <div class="modal">
+        <div class="modal-content">
+          <div class="close-button">&times;</div>
+          <ul>
+            <li><a href="#portfolio">Portfolio</a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#contact">Contact</a></li>
+          </ul>
+        </div>
```

Comment:
- [ ] Kindly check the paddings on the X and the content in order to match better the design


Yours:
![image](https://user-images.githubusercontent.com/62705947/232615540-1e4b4b00-d794-46f1-b492-f3827e194511.png)

Design:
![image](https://user-images.githubusercontent.com/62705947/232615273-62cec9d6-3b3b-4c96-8f9d-c1093d2d03ae.png)